server/src/test/java/org/elasticsearch/ingest/IngestServiceTests.java

```diff
@@ -2858,83 +2853,6 @@ public void testResolvePipelinesWithNonePipeline() {
         }
     }
 
-    public void testResolvePipelinesAndUpdateIndexRequestWithComponentTemplateSubstitutions() throws IOException {
```

Comment:
This test could be deleted because IngestService no longer knows anything about component template substitutions.